src/util.js

```diff
@@ -101,39 +102,69 @@ define([
         };
         /**
          * Register an event handler to be called each time an event is fired.
+         *
+         * @param type - Event type string.
+         * @param method - Event handler method.
+         * @param parameters - Parameters to be passed to method. If null
```

Comment:
Yeah, `parameters` came with the original code from *JavaScript: The Good Parts*. Personally I wish it wasn't there at all. It's very simple to work around it with a closure. Maybe I'll rip it out someday.